src/com/superiornetworks/icarus/modules/SeniorAdminChat.java

```diff
@@ -20,7 +20,7 @@ public static void chat(CommandSender sender, String message)
             if (ICM_Rank.isRankOrHigher(p, ICM_Rank.Rank.SENIOR))
             {
                 // Some strange ass colour codes being used here...
-                p.sendMessage("§f[§bADMIN§f] §4" + sender.getName() + " §d[SrA]§f: §d " + message);
+                p.sendMessage("§f[§bADMIN§f] §4" + sender.getName() + "[" + ICM_Rank + "] "  + message);
```

Comment:
This won't even work. You used an instance of the class itself, not one of the methods that it contains.